### PR TITLE
Migrate tests from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by slemasne**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](https://github.com/finbourne/lusid-python-tools/blob/master/docs/CONTRIBUTING.md)
- [ ] Tests pass

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

**Review by thomasdotred — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            -e FBN_USERNAME="$FBN_USERNAME" \
+            -e FBN_PASSWORD="$FBN_PASSWORD" \
+            -e FBN_CLIENT_ID="$FBN_CLIENT_ID" \
+            -e FBN_CLIENT_SECRET="$FBN_CLIENT_SECRET" \
+            -e FBN_LUSID_API_URL="$FBN_LUSID_API_URL" \
+            -e FBN_APP_NAME="$FBN_APP_NAME" \
+            -v $(pwd):/usr/src/ \
+            finbourne/lusid-python-tools-test
+          
+      - name: Run tests on DEVELOP branch
```
Comment (rizwansaeed) [inline .github/workflows/main.yml:49]: don't think we need this do we?

**Review by rizwansaeed — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (thomasdotred) [reply .github/workflows/main.yml:49]: @slemasne  this is just what I was about to ask you on the call

Comment (slemasne) [reply .github/workflows/main.yml]: Thanks, done.

File: ci_tools/bumpversion.sh [hunk 2]
```diff
@@ -19,9 +19,7 @@ else
 
     # Bump the version in project and make commit back to master
 
-    bump2version patch --message "Bump version in Travis CI: {current_version} to {new_version} [ci skip]"
-	git config --local user.email "engineering@finbourne.com"
```
Comment (rizwansaeed) [inline ci_tools/bumpversion.sh:23]: I think we still need these (https://github.com/slemasne/repo-test/commits/master)

File: ci_tools/bumpversion.sh [hunk 3]
```diff
@@ -19,9 +19,7 @@ else
```
Comment (rizwansaeed) [inline ci_tools/bumpversion.sh:19]: can you fix the spacing/tabs in this file

File: ci_tools/bumpversion.sh (hunk 2 above)
Comment (slemasne) [reply ci_tools/bumpversion.sh:23]: I had to add them directly in the GHA YAML

File: ci_tools/bumpversion.sh (hunk 3 above)
Comment (slemasne) [reply ci_tools/bumpversion.sh]: Sure, done.

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**